src/main/java/racing/domain/RacingCars.java

```diff
@@ -0,0 +1,31 @@
+package racing.domain;
+
+import racing.util.NumberGenerator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingCars {
+
+    List<RacingCar> racingCars = new ArrayList<>();
+
+    public RacingCars(int racingCarCount) {
+        createRacingCars(racingCarCount);
+    }
+
+    public void attendRound(NumberGenerator numberGenerator) {
+        for (RacingCar racingCar : racingCars) {
+            racingCar.attendRound(numberGenerator);
+        }
+    }
+
+    private void createRacingCars(int racingCarCount) {
+        for (int i = 0; i < racingCarCount; i++) {
+            racingCars.add(new RacingCar());
+        }
+    }
```

Comment:
```suggestion
    public RacingCars(int racingCarCount) {
        createRacingCars(racingCarCount);
    }

    private void createRacingCars(int racingCarCount) {
        for (int i = 0; i < racingCarCount; i++) {
            racingCars.add(new RacingCar());
        }
    }

    public void attendRound(NumberGenerator numberGenerator) {
        for (RacingCar racingCar : racingCars) {
            racingCar.attendRound(numberGenerator);
        }
    }
```

`private` 메서드는 사용되는 `public` 메서드와 그 위치를 가까이 두면 가독성에 있어 이점이 있습니다!

[오라클 자바 컨벤션 문서](https://www.oracle.com/technetwork/java/codeconventions-150003.pdf)에서 `3.1.3 Class and Interface Declarations`를 한번 읽어보시면 좋을 것 같아요!